SQL/New_patches/2021-04-26_add_annotation_data.sql

```diff
@@ -0,0 +1,143 @@
+--SQL patch to create and populate physiological annotation tables
+
+DROP TABLE IF EXISTS `physiological_annotation_instance`;
+DROP TABLE IF EXISTS `physiological_annotation_parameter`;
+DROP TABLE IF EXISTS `physiological_annotation_archive`;
+DROP TABLE IF EXISTS `physiological_annotation_file`;
+DROP TABLE IF EXISTS `physiological_annotation_file_type`;
+DROP TABLE IF EXISTS `physiological_annotation_label`;
```

Comment:
these drops need to be added to the 9999 schema

Reply:
These were added in PR #7345. As mentioned in this PR description, this is a patch that I am using to add data to the database for testing purposes and it will be deleted before merging this PR.